release-manager/files/README.md

```diff
@@ -128,8 +179,180 @@ services:
       name: leva-documentation
 ```
 
-In this case, the release manager will fall back to the document chapter templates located in the `docs` folder. Therein, you can provide chapter data to be loaded into the supported compliance documents.
+In this case, the library will fall back to the document chapter templates located in your release manager component's `docs` folder. Therein, you can provide chapter data to be loaded into the supported compliance documents.
+
+## Additional Capabilities
+
+The library supports the activation of various capabilities through the `capabilities:` field in `metadata.yml`.
+
+### Zephyr for Jira
+
+```
+capabilities:
+  - Zephyr
+```
+
+The Zephyr for Jira capability currently supports:
+
+- Reporting the result of a test execution to Zephyr for Jira
+
+## Environment Promotion
+
+This section will guide you through the "environment promotion" feature of the orchestration pipeline.
+It is assumed have the release manager quickstarter already provisioned and configured in your project.
```

Comment:
typo?
It is assumed to have....

Reply:
well, as it is a copy-paste of the document on the shared library this line is there. we could remove it on this side